Features.md

```diff
@@ -0,0 +1,48 @@
+
+## Support status
+
+This is a (non-exhaustive) list of features currently (un)supported by CHIP-SPV.
+
+### Host side
+
+#### Unsupported / unimplemented APIs
+
+* hipRtc API
+* hipGraph API
+* hipIpc API
+* hipModuleOccupancy API
+* hipTexRef (texture reference) API
+* surface object / reference APIs
+* hipMemcpyPeer, hipMemcpyPeerAsync, hipMemRangeGetAttribute,
+  hipDeviceSetCacheConfig, hipDeviceGetCacheConfig,
+  hipDeviceSetSharedMemConfig, hipDeviceGetSharedMemConfig,
+  hipSetDeviceFlags,  hipGetDeviceFlags,
+  hipDeviceCanAccessPeer, hipDeviceEnablePeerAccess,
+  hipDeviceDisablePeerAccess, hipDeviceGetStreamPriorityRange,
+  hipStreamGetPriority, hipDevicePrimaryCtxRelease, hipDevicePrimaryCtxRetain,
+  hipDevicePrimaryCtxSetFlags, hipMemPrefetchAsync, hipMemAdvise,
+  hipModuleLoadData, hipModuleUnload, hipModuleLaunchKernel, hipStreamPerThread
+
+#### partially supported
+  * Texture Objects of 1D/2D type are supported; 3D, LOD, Grad,
+    Cubemap, Gather and Mipmapped textures are not supported
+
+### Device side
+
+#### Unsupported / unavailable
+* __syncwarp(), __activemask()
+* cooperative_groups.h header
+* Address Space Predicate Functions, Address Space Conversion Functions
+* alloca(), malloc(), free()
+* Warp Reduce Functions, Warp Matrix Functions
```

Comment:
Seems like too short of a list given the number of compilation unit tests that fail.

Reply:
yeah it's not a full list, also i'm still working on fixing some math library functions (there's a rint/round fix in next PR).